### ISSUE TITLE
what fonts do you use?

### ISSUE THREAD

**Issue by RobinHirst11** [thytom/dwmbar#38]
hello! i currently have ttf-font-awesome and ttf-jetbrains-mono installed, it works for all apart from localip

do you mind sharing what font is best for this?

**Comment by thytom** [comment]
Hello!

It's been a few years since I've used this myself but I would always recommend [Nerd Fonts](https://www.nerdfonts.com/). It bundles a load of icons including fontawesome together and patches popular fonts with them.

A lot of people like inconsolata, I use GoMono. If you're on arch, I'd avoid the AUR packages and instead install them manually in `.local/share/fonts` and calling `fc-cache` as https://wiki.archlinux.org/title/Fonts suggests.

**Comment by thytom** [comment]
As an aside, you can also patch other fonts, like jetbrains, with the nerdfont icons manually. I've not done this in a long time though. Check out https://github.com/ryanoasis/nerd-fonts for more details on that.

**Comment by RobinHirst11** [comment]
yeah just because the localip module uses an unknown symbol so.. idk

**Comment by RobinHirst11** [comment]
apparently the prefix is ﯱ

**Comment by thytom** [comment]
Huh. Why that's the case I have no idea. Could have been a fontawesome icon which turns out is something else when using other fonts? 

Unicode supports "ᯤ" which seems like a sensible default option, so I'll change it to that.
